Geometry/Fields/VectorFields/Tests/weightWindowsField_test.f90

```diff
@@ -0,0 +1,87 @@
+module weightWindowsField_test
+  use numPrecision
+  use pFUnit_mod
+  use particle_class,           only : particle
+  use dictionary_class,         only : dictionary
+  use dictParser_func,          only : charToDict
+  use weightWindowsField_class, only : weightWindowsField
+
+  implicit none
+
+
+@testCase
+  type, extends(TestCase) :: test_weightWindows
+    private
+    type(weightWindowsField) :: wwField
+  contains
+    procedure :: setUp
+    procedure :: tearDown
+  end type test_weightWindows
+
+
+  !!
+  !! Weight Windows Definition
+  !!
+  character(*), parameter :: DICT_DEF = &
+  & "file /home/vr339/SCONE/scone/IntegrationTestFiles/testWW ;"
```

Comment:
If it is using a file it needs to become an 'Integration Test' (with `_iTest` suffix). Also path should be relative to the git root directory i.e. `IntegrationTestFiles/testWW ;`